perfkitbenchmarker/requirements.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions for checking that required Python packages are installed."""
+
+import os
+import sys
+
+import pkg_resources
+
+from perfkitbenchmarker import errors
+
+
+_DEFAULT_BRANCH_ROOT_DIR = sys.path[0]
```

Comment:
This seems a bit magical to me. I think path relative to this file would be clearer.